tests/testsuite/directory.rs

```diff
@@ -1,5 +1,5 @@
 use ansi_term::Color;
-use dirs::home_dir;
+use dirs_next::home_dir;
```

Comment:
This one is fine and arguably should have been in here in the first place, since we use it so often :+1: